includes/class-freemius.php

```diff
@@ -5426,6 +5516,10 @@ function is_ssl_and_plan( $plan, $exact = false ) {
 		 * @return string
 		 */
 		function _get_admin_page_url( $page = '', $params = array() ) {
+			if ( ! $this->_plugin->is_plugin() ) {
```

Comment:
@fajardoleo Some themes might have settings menu, if that's the case (when there's a menu params set using the dynamic_init), it should work similarly like with plugins. The redirect should go to the settings page, and if the user is in activation / opt-in mode, it should override the default settings (like in plugins) with the opt-in page.